packages/components/src/TemporaryAddressChange/Component.purs

```diff
@@ -266,10 +269,18 @@ render self@{ state: { startDate, endDate, streetAddress, zipCode, countryCode,
                                    , temporaryName: temporaryName'
                                    } = do
           liftEffect $ self.props.onLoading
+          let startDate''  = formatDate startDate'
```

Comment:
Let's not call these `startDate''` and `endDate''`, but `startDateString` and `endDateString`